src/dialogs/transcodedialog.ui

```diff
@@ -159,31 +163,34 @@
           </property>
          </widget>
         </item>
-        <item row="2" column="1">
-         <widget class="FrameRateWidget" name="fpsWidget" native="true">
+        <item row="3" column="1">
+         <widget class="QComboBox" name="frcComboBox">
           <property name="toolTip">
-           <string>Override the frame rate to a specific value.</string>
+           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Frame Rate Conversion Method&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Duplicate: &lt;/span&gt;Duplicate frames&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Blend: &lt;/span&gt;Blend frames&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Motion Compensation: &lt;/span&gt;Interpolate new frames using motion compensation. This method is very slow and may result in artifacts.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>

```

Comment:
I do not like this rich text tooltip because it makes translation difficult or easy to break. Translation UIs often do not provide a rich text editor, and they can easily break the HTML or simply skip over it as "too much work; maybe sometime later." Maybe it is better to keep the tooltip simple and have more documentation in the forum docs category.

Reply:
Done